official/resnet/resnet_run_loop.py

```diff
@@ -402,8 +404,16 @@ def resnet_main(flags_obj, model_function, input_function, shape=None):
           'dtype': flags_core.get_tf_dtype(flags_obj)
       })
 
+  hyperparams = {
+      'batch_size': flags_obj.batch_size,
+      'dtype': flags_obj.dtype,
```

Comment:
I think this should be `flags_core.get_tf_dtype(flags_obj).name` That way we're logging the official name rather than whatever abbreviation we choose.

Reply:
Done.